package.json

```diff
@@ -25,6 +25,7 @@
   "scripts": {
     "prebuild": "npm run lint",
     "build": "babel src --out-dir dist",
+    "prepare": "npm run build",
```

Comment:
Ah, good call. I probably should just drop the "prepublish" script altogether in place of the "prepare" script.